libs/designsystem/src/lib/components/range/range.component.scss

```diff
@@ -0,0 +1,86 @@
+@import '../../helpers/color-helper';
+@import '../../scss/utils';
+
+$tickHeight: 6px;
+$tickWidth: 6px;
+
+:host {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+ion-range {
+  --knob-background: #{get-color('white')};
+  --knob-box-shadow: 0 5px 10px 0px rgba(28, 28, 28, 0.3), 0 0 5px 0 rgba(28, 28, 28, 0.08);
+  --knob-size: 24px;
+
+  --pin-color: #{get-text-color('black')};
+  --pin-background: #{get-color('semi-light')};
+
+  --bar-background: #{get-color('medium')};
+  --bar-background-active: #{get-color('dark')};
+  --bar-border-radius: #{size('xxxs')};
+  --bar-height: #{size('xxxxs')};
+
+  &.range-disabled {
+    --knob-background: #{get-color('semi-light')};
+    --knob-box-shadow: none;
+
+    --bar-background: #{get-color('semi-light')};
+    --bar-background-active: #{get-color('semi-light')};
+
+    &::part(tick) {
+      background: get-color('semi-light');
+    }
+  }
+
+  flex: 1 1 100%;
+  padding: 0;
+
+  &.range-has-pin {
+    padding: 0 $tickWidth / 2;
+  }
+
+  &::part(pin) {
+    font-size: font-size('xs');
+    color: var(--pin-color);
+    background-color: var(--pin-background);
+    padding: size('xxxxs') size('xxxs');
+    border-radius: 4px;
+    min-width: 40px;
+    position: relative;
+
+    &:before {
+      content: '';
+      position: absolute;
+      top: 100%;
+      left: 50%;
+      transform: translateX(-50%);
+      display: block;
+      border-color: var(--pin-background) transparent transparent transparent;
+      border-style: solid;
+      border-width: 6px;
```

Comment:
Magic number - I just tested with `size('xxxl')`. It looks fine, if the spacing is adjusted a bit IMO

Reply:
Sizes currently have the comment "spacings", so I'm not sure if it would be correct to use it for everything?